Components/TeamList.js

```diff
@@ -0,0 +1,68 @@
+import useSWR from "swr";
+import styled from "styled-components";
+import Image from "next/image";
+
+const Header = styled.h1`
+  font-size: 5rem;
+  font-weight: bold;
+  margin-top: 1rem;  
+  font-family: "roboto", sans-serif;
+  text-align: center;
+  letter-spacing: 3px;
+ 
+`;
```

Comment:
One tiny note 🤏
`Headline` or `Title` would be a slightly more descriptive name for your component. 

```suggestion
const Title = styled.h1`
  font-size: 5rem;
  font-weight: bold;
  margin-top: 1rem;  
  font-family: "roboto", sans-serif;
  text-align: center;
  letter-spacing: 3px;
 
`;
```